config/locales/views/peer_reviews/en.yml

```diff
@@ -9,6 +9,9 @@ en:
     assigned_reviewers_header: Reviewers
     errors:
       cannot_allow_reviewer_to_be_reviewee: A student cannot be a reviewer and reviewee
+      cannot_unassign_all_reviewers: 'Cannot unassign all selected peer reviewers due to existing marks or annotations. %{deleted_count}  peer reviewer(s) were unassigned, but the following could not be: %{undeleted_reviews}.'
```

Comment:
Reword to "Successfully unassigned %{deleted_count} peer reviewer(s), but could not unassign the following due to existing marks or annotations: ..."